charts/kubescape-operator/templates/node-agent/configmap.yaml

```diff
@@ -14,9 +14,28 @@ data:
     {
         "applicationProfileServiceEnabled": {{ $configurations.runtimeObservability }},
         "relevantCVEServiceEnabled": true,
+        "prometheusExporterEnabled": {{ .Values.nodeAgent.serviceMonitor.enabled }},
+        "runtimeDetectionEnabled": {{ eq .Values.capabilities.runtimeDetection "enable" }},
+        "networkServiceEnabled": {{ eq .Values.capabilities.networkPolicyService "enable" }},
+        "malwareDetectionEnabled": {{ eq .Values.capabilities.malwareDetection "enable" }},
```

Comment:
@dwertent TODO: update node agent tag which supports this new config

Reply:
Done